### PR TITLE
[15.0][MIG] base import async: Migration to 15.0

### PR DESCRIPTION
**Pull request by ieski**

### CODE REVIEW

**Comment by ieski** [comment]
@dhongu can you review it?
Thanks

**Comment by dzungtran89** [comment]
Good day, could you please let me know when this PR can be merged? I need it for the 16.0 migration

**Review by dzungtran89 — APPROVED** [review]

**Comment by ChrisOForgeFlow** [comment]
@ieski Can you rebase please, and check pre-commit warnings, to continue with review, Thank you

File: base_import_async/__manifest__.py [hunk 1]
```diff
@@ -5,14 +5,23 @@
 {
     "name": "Asynchronous Import",
     "summary": "Import CSV files in the background",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (dzungtran89) [inline base_import_async/__manifest__.py:8]:
```suggestion
    "version": "15.0.1.0.0",
```

Comment (dzungtran89) [reply base_import_async/__manifest__.py:8]: apart of this and @ChrisOForgeFlow's comment, lgtm

**Pull request closed without merging**